openwisp_radius/api/views.py

```diff
@@ -27,6 +27,7 @@
 from rest_framework import generics, serializers, status
 from rest_framework.authentication import BaseAuthentication, SessionAuthentication
 from rest_framework.authentication import TokenAuthentication as UserTokenAuthentication
+from openwisp_users.api.authentication import BearerAuthentication
```

Comment:
BatchView uses `BearerAuthentication` but others use `UserTokenAuthentication`; I think all of them should be moved to `BearerAuthentication`, should I make a PR for it? :smile:

Reply:
Nevermind, found https://github.com/openwisp/openwisp-radius/issues/91 ; I'll get to that as soon as possible! :smile: